public/localization/en-US.js

```diff
@@ -295,7 +295,8 @@ module.exports = {
         TEMPLATE: 'Template',
         HEADLINE: 'Headline',
         SUBHEADING: 'Subheading',
-        PUBLISH_DATE: 'Publish date'
+        PUBLISH_DATE: 'Publish date',
+        PAGE_NAME: 'Page name'
```

Comment:
I was not sure of the approval of the idea. I think if idea will approved then i add this values to all localization files.